java-squid/src/test/files/se/UnclosedResourcesCheck.java

```diff
@@ -240,6 +248,18 @@ public void correctHandlingOfJarFile(String fileName) {
       closeJar(jar);
     }
   }
+
+  public void getDirectivesFromFile(File aFile) {
+    BufferedReader reader = null;
+    try {
+      reader = new BufferedReader(new FileReader(aFile));
+      reader.read();
+//    } catch(IOException e) { // Causes a problem when decommented!
```

Comment:
What do you mean? This is just test code copied verbatim from a file from "ruling"...